credsweeper/file_handler/data_content_provider.py

```diff
@@ -32,6 +43,41 @@ def data(self, data: bytes) -> None:
         """data setter"""
         self.__data = data
 
+    def is_xml(self) -> bool:
+        """Tries to read data as xml
+        Return True if reading was successful
+        """
+        try:
+            xml_text = self.data.decode(encoding=DEFAULT_ENCODING).splitlines()
+            tree = etree.fromstringlist(xml_text)
+            for element in tree.iter():
+                tag = Util.extract_element_data(element, "tag")
+                text = Util.extract_element_data(element, "text")
+                self.lines.append(f"{tag} : {text}")
+        except Exception as exc:
+            logger.debug("Cannot parse as XML:%s %s", exc, self.data)
+            return False
+        return bool(self.lines)
+
+    def is_encoded(self) -> bool:
+        """Encodes data from base64. Stores result in decoded
+
+        Return:
+             True if the data correctly parsed and verified
+
+        """
+        if len(self.data) < 12 or (b"=" in self.data and b"=" != self.data[-1]):
+            logger.debug("Weak data to decode from base64: %s", self.data)
+        try:
+            self.decoded = base64.b64decode(  #
+                self.data.decode(encoding='ascii', errors='strict').  #
+                translate(str.maketrans('', '', string.whitespace)),  #
+                validate=True)  #
```

Comment:
Changing single quote to double quote.

```suggestion
        try:
            self.decoded = base64.b64decode(  #
                self.data.decode(encoding="ascii", errors="strict").  #
                translate(str.maketrans("", "", string.whitespace)),  #
                validate=True)  #
```